infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesWorkspaceServiceAccount.java

```diff
@@ -49,9 +49,6 @@ public KubernetesWorkspaceServiceAccount(
   /**
    * Make sure that workspace service account exists and has `view` and `exec` role bindings.
```

Comment:
Please consider sync this java doc with `OpenShiftWorkspaceServiceAccount#prepare`

Reply:
Fixed